scripts/verify_graph.py

```diff
@@ -135,15 +346,23 @@ def fetch_graph_ids():
 
 exit_code = 0
 for game_version in fetch_graph_ids():
-    print("Check edge lenghts for game version " + game_version + ":")
+    print("Check graphs for game version " + game_version + ":")
     with open(os.path.join(graph_path, "ss_" + game_version + "_graph.json")) as graph_file:
         graph_data = json.load(graph_file)
     
     with open(os.path.join(raw_path, "ss_" + game_version + "_raw.json")) as graph_file:
         raw_data = json.load(graph_file)
 
+    # Checks all edges up against raw path data.  This is the simple check
     if check_edge_lengths(graph_data, raw_data, warning_prefix):
-        print(" - OK")
+        print(" - Current edge lenghts: OK")
+    else:
+        exit_code = 1
+
+    # Checks all possible vertex-connection combinations (via one or more
+    # edges) up against raw path data.  This is the more complex check
```

Comment:
`vertex-connection combinations` = `combinations of verticies`. Plus remove double white space (in the above comment as well